x-pack/plugins/security/public/views/management/roles.html

```diff
@@ -1,228 +0,0 @@
-                i18n-default-message="Create role"
-              ></span>
-            </a>
-          </div>
-
-          <div class="kuiToolBarSection">
-            <!-- We need an empty section for the buttons to be positioned consistently. -->
-          </div>
-        </div>
-
-        <!-- NoResults -->
-        <div class="kuiPanel kuiPanel--centered" ng-if="!(roles | filter:query).length">
-          <div
-            class="kuiNoItems"
-          >
-            <span
-              ng-if="query"
-              i18n-id="xpack.security.management.roles.noMatchingResultsLabel"
-              i18n-default-message="No matching roles found"
-            ></span>
-            <span
-              ng-if="!query"
-              i18n-id="xpack.security.management.roles.noResultsLabel"
-              i18n-default-message="No roles found"
-            ></span>
-          </div>
-        </div>
-
-        <!-- Table -->
-        <table class="kuiTable" ng-show="(roles | filter:query).length">
-          <thead>
-            <tr>
-              <th scope="col" class="kuiTableHeaderCell kuiTableHeaderCell--checkBox">
-                <div class="kuiTableHeaderCell__liner">
-                  <input
-                    type="checkbox"
-                    class="kuiCheckBox"
-                    ng-checked="allSelected()"
-                    ng-click="toggleAll()"
-                    aria-label="{{allSelected() ? 'Deselect all rows' : 'Select all rows'}}"
-                  >
-                </div>
-              </th>
-
-              <th
-                class="kuiTableHeaderCell"
-                scope="col"
-              >
-                <button
-                  class="kuiTableHeaderCellButton"
-                  ng-class="{'kuiTableHeaderCellButton-isSorted': sort.orderBy == 'name'}"
-                  ng-click="toggleSort(sort, 'name')"
-                  aria-label="{{sort.reverse ? 'Sort role ascending' : 'Sort role descending'}}"
-                >
-                  <span class="kuiTableHeaderCell__liner">
-                    <span
-                      i18n-id="xpack.security.management.roles.roleTitle"
-                      i18n-default-message="Role"
-                    ></span>
-                    <span
-                      aria-hidden="true"
-                      class="kuiTableSortIcon kuiIcon"
-                      ng-class="getSortArrowClass('name')"
-                    ></span>
-                  </span>
-                </button>
-              </th>
-
-              <th scope="col" class="kuiTableHeaderCell">
-                <span
-                  class="kuiTableHeaderCell__liner"
-                  i18n-id="xpack.security.management.roles.reversedTitle"
-                  i18n-default-message="Reserved"
-                ></span>
-                <span class="kuiIcon fa-question-circle" tooltip={{reversedTooltip}} aria-label={{reversedAriaLabel}} ></span>
```

Comment:
Should we add this tooltip back in as well?

<img width="267" alt="Screen Shot 2019-04-05 at 13 09 59 PM" src="https://user-images.githubusercontent.com/549577/55657259-a59aba00-57c7-11e9-9e91-2e6d03449908.png">